### DISCUSSION TITLE
I want to achieve voice cloning. Where can I download the model?

### DISCUSSION THREAD

**Discussion by bbhxwl** [coqui-ai/TTS#4068] [General Q&A]
where tts_models/en/vctk/vits ?
```
from TTS.api import TTS

# 加载支持语音克隆的模型
tts = TTS(model_name="tts_models/en/vctk/vits", progress_bar=False, gpu=False)

# 输入文本
text = "Hello, this is a cloned voice speaking!"

# 提供目标语音样本路径
voice_sample_path = "path/to/voice_sample.wav"

# 输出语音保存路径
output_path = "output_cloned_voice.wav"

# 生成语音并保存
tts.tts_to_file(text=text, speaker_wav=voice_sample_path, file_path=output_path)

print(f"Cloned voice TTS saved to {output_path}")
```

**Comment by eginhard** [comment]
That model doesn't support voice cloning. You can use YourTTS if you want a Vits-based model, otherwise XTTS.

**Reply by JacosNet** [reply]
Cloning of the sample voice is also not working for me, here is the code:

`from TTS.api import TTS`
`tts = TTS("tts_models/multilingual/multi-dataset/xtts_v2")`
`tts.tts_to_file(
    text="My name is Alex, and I'm now on the way to school! I'm going to be talking about some really cool ideas for building colonies on Mars. We're planning to start our journey next year and build an amazing team with our friends. If you're interested in joining us or have any questions, please don't hesitate to reach out. I'm always happy to help! How can I help you today?",
    speaker_wav="my/sample.wav",
    language="en",
    file_path="output/xtts_v2.wav"
)`

Its result is a mix of different speakers they are speaking the parts of the string, but none of them is the clone of my sample voice.

When I try to use the model yout_tts then the result is same but only with one speaker, again nothing is cloned.

What is wrong in my code, please not, I'm using Mac M1 without GPU.

**Reply by eginhard** [reply]
Which version do you have? There was indeed a bug with inconsistent voice cloning in coqui-tts 0.25.0, but it should be normal in other versions. For better quality, you could fine-tune the model.

**Comment by JacosNet** [comment]
I've installed the last version two weeks ago. But, as I see, the last version is v0.22.0 (Dec 12, 2023), why are you mentioning 0.25.0?

What about the mixed speakers, is there something parameters missing in my code?

Could you provide me the link how to fine-tune the model?

**Reply by eginhard** [reply]
Ok, then it's just random variation of the model. For fine-tuning: https://coqui-tts.readthedocs.io/en/latest/models/xtts.html#advanced-training

> I've installed the last version two weeks ago. But, as I see, the last version is v0.22.0 (Dec 12, 2023), why are you mentioning 0.25.0?

This repo is abandoned, but we continue to maintain a fork here: https://github.com/idiap/coqui-ai-TTS

**Comment by JacosNet** [comment]
Thanks, after update it's working as expected
